### PR TITLE
added an Armstrong solution

### PR DESCRIPTION
**Pull request by Barno74**
Please review ASAP

### CODE REVIEW

**Review by tilbearslan — APPROVED** [review]
Great

**Comment by Barno74** [comment]
Please review ASAP.Thank you

**Review by nurgul3374 — COMMENTED** [review]
Approved

**Review by Gulynim — APPROVED** [review]
Great!

File: src/barno/ArmstrongNumber.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    int result=0;
+    int remainder=0;
+    int originalNumber=number; //Math.pow(a,n)=a*a*....n times;
+
+        while(originalNumber!=0){ //while the number is not 0, 15!=0, 1!=0
+        remainder=originalNumber%10; // remainder 3==>15%10=>remainder=>5==1%10=>remainder =1
+        result+=Math.pow(remainder,length); //armstrong number is 3, because 3 digits:3*3*3=27;5*5*5=125;1*1*1=1
+        //27+125+1=153
+        originalNumber=originalNumber/10;//153/10=>number=15;15/10=1;1/10=0
+
```
Comment (Gulynim) [inline src/barno/ArmstrongNumber.java:19]: Looks good

**Review by Gulynim — COMMENTED** [review]
GG

**Comment by aiturgank** [comment]
GLTM!

**Pull request merged**